packages/lab/src/stepped-tracker/TrackerStep/TrackerStep.tsx

```diff
@@ -19,46 +21,34 @@ import trackerStepCss from "./TrackerStep.css";
 
 const withBaseName = makePrefixer("saltTrackerStep");
 
-type State = "default" | "completed";
+type StageOptions = "pending" | "completed";
+type StatusOptions = Extract<ValidationStatus, "warning" | "error">;
 
-type StateWithActive = State | "active";
+interface ParseIconNameProps {
```

Comment:
Can we inline this type where it's used? As far as I can tell its not shared and it's extra work to check the types like this